### PR TITLE
Hide history tab when table is empty

### LINKED ISSUE #108: Hide history tab when the history is empty.
As detailed by the product team.

### PR DESCRIPTION
**Pull request by carlosms**
Fixes #108.

### CODE REVIEW

**Comment by carlosms** [comment]
Yes, you can click 'reset history' at any time.

File: frontend/src/components/TabbedResults.less [hunk 1]
```diff
@@ -68,9 +68,11 @@
         font-weight: @bold-font-weight;
     }
 
-    .nav-tabs {
-        > li:last-child {
-            float: right;
+    &.has-history {
```
[the remark this answers is not in the document]
Comment (carlosms) [reply frontend/src/components/TabbedResults.less]: The problem is that you can set the html that goes inside the tab title `<li><a>`, but what needs to float, the `<li>`, is created by bootstrap. I thought this was the cleanest option.

[the remark this answers is not in the document]
Comment (smacker) [reply frontend/src/components/TabbedResults.less]:
```
          <Tab
            key="history"
            eventKey="history"
            tabClassName="qwe"
```

screen shot 2018-06-19 at 11 20 57

does it work for you?

[the remark this answers is not in the document]
Comment (carlosms) [reply frontend/src/components/TabbedResults.less]: I wasn't aware of `tabClassName`, that makes it cleaner :+1: 

Suggestion applied, rebased and squashed. Will merge now.

**Pull request merged**